integration/test-integration/pom.xml

```diff
@@ -44,6 +44,14 @@
                     <suiteXmlFiles>
                         <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
                     </suiteXmlFiles>
+                    <systemProperties>
+                        <property>
+                            <name>framework.resource.location</name>
+                            <value>
+                                ${basedir}/src/test/resources/
```

Comment:
This location can be accessed using the path as [TestConstant.TEST_RESOURCES_PATH](https://github.com/wso2/product-microgateway/blob/9509397ca5e9a886815b8759c342e13a52867ba3/integration/test-integration/src/test/java/org/wso2am/micro/gw/tests/util/TestConstant.java#L75). An example can be found here: https://github.com/wso2/product-microgateway/blob/9509397ca5e9a886815b8759c342e13a52867ba3/integration/test-integration/src/test/java/org/wso2am/micro/gw/tests/util/ApictlUtils.java#L50

Reply:
This is just setting a system property called "framework.resource.location" (inside the pom.xml), so that the automation engine can configure the automation.xml before running the apim related integration tests.

Reply:
Got it. Thanks. +1